lib/rules/require-template-strings.js

```diff
@@ -62,28 +64,8 @@ module.exports.prototype = {
                 return;
             }
 
-            // One of the operands should be a string otherwise this is not a concatination
-            if (typeof node.left.value !== 'string' && typeof node.right.value !== 'string') {
-                return;
-            }
-
-            // "a" + a
-            if (node.right.type === 'Identifier') {
-                add(node);
-            }
-
-            // a + "a"
-            if (node.left.type === 'Identifier') {
-                add(node);
-            }
-
-            // (a + b) + "a"
-            if (node.left.type === 'BinaryExpression') {
-                add(node);
-            }
-
-            // "a" + (a + b)
-            if (node.right.type === 'BinaryExpression') {
+            // Only one of the operands should be a string, otherwise this is not a concatenation
+            if (typeof node.left.value === 'string' || typeof node.right.value === 'string') {
                 add(node);
```

Comment:
Will that fix the coverage issue? We still need the curly braces since we have that rule on.